packages/kbn-profiling-utils/common/stack_traces.ts

```diff
@@ -111,15 +111,21 @@ export const makeFrameID = (frameID: string, n: number): string => {
 // createInlineTrace builds a new StackTrace with inline frames.
 const createInlineTrace = (
   trace: ProfilingStackTrace,
-  frames: Map<StackFrameID, StackFrame>
+  frames: Map<StackFrameID, StackFrame>,
+  showErrorFrames: boolean
 ): StackTrace => {
   // The arrays need to be extended with the inline frame information.
   const frameIDs: string[] = [];
   const fileIDs: string[] = [];
   const addressOrLines: number[] = [];
   const typeIDs: number[] = [];
 
-  for (let i = 0; i < trace.frame_ids.length; i++) {
+  // Error frames only appear as first frame in a stacktrace.
+  const start =
+    // eslint-disable-next-line no-bitwise
+    !showErrorFrames && trace.frame_ids.length > 0 && (trace.type_ids[0] & 0x80) !== 0 ? 1 : 0;
```

Comment:
I see this exact same logic is defined here: https://github.com/elastic/kibana/pull/176537/files#diff-7ad6cb41704ade392bb935e6b22635de37df126f551a2676bb540d29221c990fR109

Can't you export to a utils file and reuse it?

Reply:
Created `isErrorFrame()`.